### ISSUE TITLE
Some form integrations are calling both AJAX and PHP submission handlers

### ISSUE THREAD

**Issue by fpcorso** [PopupMaker/Popup-Maker#887]
## Describe the bug
Some of our form integrations, including CF7 and WPForms, are calling both our AJAX and PHP submission handlers resulting in duplicate processes for cookie creation and conversion counts.

## Site information
Popup Maker version: 1.12.0
WordPress version: 5.5.1
PHP version: 7.4.1

## Proposed Solution
We should have a method in the abstract form integration class that will check if it's an AJAX or REST call. We then use that method within each form integration to determine if we should process the form submission.

**Comment by danieliser** [comment]
I think the best option here is to add an extra value to the `pum_vars` and then use that to bypass a beacon hits.

**Comment by fpcorso** [comment]
@danieliser I think you confused this issue with the other one relating to PHP submission sending data to the front-end, which has already been resolved.

This would not be solved by any `pum_vars` as the AJAX request happens first. Remember, in our Slack thread, I explained that some of the form integrations call both submission handlers when submitted via AJAX. Refer to our thread here: https://app.slack.com/client/T1Q9SV610/GP01LEMM1/thread/GP01LEMM1-1602605298.007200

So, during the form submission, it currently goes:
1. User clicks submit
2. JS fires form success, which creates cookie and fires conversion counter
3. PHP process form submission, firing our form success method, which creates cookie and fires conversion counter

The solution that you proposed in Slack is the one I listed in this issue.
